setup-scripts/support/authorized_keys

```diff
@@ -9,3 +9,4 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN6Kcx8yetova4/ALUQHigo/PBMJO33ZTKOsg2jxSO2a
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDcenWnMQ6q/OEC4ZmQgjLDV2obWlR3fENV0zRGFvJF+ marcins@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIB7prtVOTwtcSN9HkXum107RwcW5H8Vggx6Qv7T57ItT daniel@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIH+4IrfEM9H16jqvPZncHkWWoHO4/BVq7d4pEyzK4e0W michal.leszczyk@skynetlabs.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDY48P1s8NAgjKFRN2/iUYXWbYsEb10kyPRPxlawtI4JqEMfDxan30c99ANe6badh3KS+EDLdviijD7FDKGP2sPSBWv4L0N996u5t8+tK+eon3kRaJfGfyHlAuQWoRIHU1IK/liMqcJ5RgpZxavwMYbz0nza3LVRgkzfAR4qtWOBL3TfWclRHqYC6+qvP9Y3mc/FJxXPCyFBukZt4z3kFSW4WzoZuPxubw7wfOhgiG+2NSjLMxmKbFOCqcDfD/UyAY/z3JY9VSOBqX2+XqTex4O0WYNt4GaVd3Spo5sWV2s+xAPnBLEr4+pN45reLs2CaJjjqygPWDPJ8fDUjK1qSgAmfOFiP79jDDsnU/+fL0HsnmfmVC79td5gwi7TTmoAGSXzCNKCgcW5Q1GecSdG2doXGuNYfcvX5S737K+9KU+ril5rUKxvNjHWugYStbgDU0xvKtI5buI+CYD+p2OP0qRupb9jmwuZ+EsIwSX09LzITjs0H9sZ3Jd36eTT9EmHPc= user@dev
```

Comment:
Didn't we want everyone to switch to ssh-ed25519 ?

https://levelup.gitconnected.com/demystifying-ssh-rsa-in-openssh-deprecation-notice-22feb1b52acd#:~:text=OpenSSH%20will%20be%20deprecating%20the,less%20than%20USD%2450K.

Reply:
```suggestion
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHptEpqs57lhnHkfa+0SQgXQ4A63/YGV2cNTcGMQW+Jt david@skynetlabs.com
```